Books/server/src/index.ts

```diff
@@ -27,9 +28,11 @@ app.get('/', (req: Request, res: Response) => {
   })
 })
 
+app.use(errorMiddleware)
+
 // start express server
```

Comment:
Is this comment really necessary? I think the app.listen() method is pretty self explantory, what do you think?